tests/tests/test-precompile/test-precompile-staking.ts

```diff
@@ -137,6 +137,32 @@ describeDevMoonbeamAllEthTxTypes("Precompiles - Staking - Join Delegators", (con
     ).to.equal(alith.address, "delegation didn't go through");
     expect(delegatorsAfter.status.toString()).equal("Active");
   });
+
+  it("should have correct delegation amount for ethan to ALITH", async function () {
+    // Check that delegation amount equals MIN_GLMR_STAKING
+    const { result } = await web3EthCall(context.web3, {
+      to: PRECOMPILE_PARACHAIN_STAKING_ADDRESS,
+      data: PARACHAIN_STAKING_INTERFACE.encodeFunctionData("delegationAmount", [
+        ethan.address,
+        alith.address,
+      ]),
+    });
+
+    expect(BigInt(result)).to.equal(MIN_GLMR_STAKING);
```

Comment:
@4meta5 if you compare to bigint number it is required to use BigInt(...) and not Number(...)
I fixed the test